plugins/woocommerce-admin/client/tasks/tasks.scss

```diff
@@ -239,8 +239,10 @@
 }
 
 .woocommerce-task-list__setup {
-	.woocommerce-experimental-list .woocommerce-experimental-list__item.complete {
+	.woocommerce-experimental-list
+	.woocommerce-experimental-list__item.complete {
 		text-decoration: line-through;
+		color: var(--wp-admin-theme-color);
 
 		.woocommerce-task-list__item-title {
 			color: var(--wp-admin-theme-color);
```

Comment:
Nitpick: I think the rule in `.woocommerce-task-list__item-title` is redundant and can be removed now it's in the parent

Reply:
Thank you for the suggestion! I tested it after removing `color`, but it did not work. There are other places where we define color for `.woocommerce-task-list__item-title` selector. We need to explicitly define it here to make sure the theme color applies.